.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Report a problem with this library that needs to be fixed.
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**Before filing this issue**
+Make sure you've carefully reviewed the
+[documentation](https://developers.google.com/google-ads/api/docs/client-libs/python)
+for this library.
+
+This library is only used for the
+[Google Ads API](https://developers.google.com/google-ads/api/docs/start), for
+the [AdWords API](https://developers.google.com/adwords/api/docs/guides/start)
+client library report your issue
+[here](https://github.com/googleads/googleads-python-lib/issues).
+
+Only use this page to report problems with this library. If you are unsure
+whether your issue is related to this library or the Google Ads API itself,
+please first reach out to one of the below resources before filing an issue here:
+
+* [Post in the Support Forum](https://support.google.com/google-ads/thread/new)
```

Comment:
The link we use with blog posts for the forum is https://groups.google.com/forum/#!forum/adwords-api

Reply:
Done